app/views/DR/Aurora/index.js

```diff
@@ -1,18 +1,11 @@
-import React, { useEffect } from 'react';
-import { View, TouchableOpacity } from 'react-native';
 import { Text } from 'native-base';
-import { WebView } from 'react-native-webview';
+import React, { useEffect } from 'react';
```

Comment:
This file does is not necessary for the report

Reply:
I think he added other things besides the report